operator/controllers/loki/lokistack_controller.go

```diff
@@ -224,3 +235,31 @@ func (r *LokiStackReconciler) enqueueAllLokiStacksHandler() handler.EventHandler
 		return requests
 	})
 }
+
+func (r *LokiStackReconciler) enqueueForUserWorkloadAMService() handler.EventHandler {
+	ctx := context.TODO()
+	return handler.EnqueueRequestsFromMapFunc(func(obj client.Object) []reconcile.Request {
+		lokiStacks := &lokiv1.LokiStackList{}
+		if err := r.Client.List(ctx, lokiStacks); err != nil {
+			r.Log.Error(err, "Error getting LokiStack resources in event handler")
+			return nil
+		}
+		var requests []reconcile.Request
+
+		if obj.GetName() == openshift.MonitoringSVCOperated && obj.GetNamespace() == openshift.MonitoringUserwWrkloadNS {
```

Comment:
What about the AM in the `openshift-monitoring` namespace?

Reply:
what about it?

Reply:
Don't we need to check for it here?

Reply:
maybe, maybe not, but this is not the PR for that

Reply:
Why not?

Reply:
I'd rather get this one right and merged then do another PR for the other AM.

Reply:
I see this being the same for both AM as both services have the name `alertmanager-operated` but differ only in a namespace.

Reply:
:shrug: